scripts/release/homebrew/docker/requirements.txt

```diff
@@ -1,4 +1,4 @@
 homebrew-pypi-poet~=0.10.0
-jinja2==3.1.5
+jinja2~=3.1.6
```

Comment:
There are so many PR about jinja2 version. Unpin it.
![image](https://github.com/user-attachments/assets/27f61cf6-1c33-4728-a202-2c768c224c2c)